.travis.yml

```diff
@@ -22,3 +22,4 @@ script:
   - make check-license
   - make test
   - make tools
+  - sudo make install
```

Comment:
do we need to have sudo here, if we just add a DESTDIR variable so it can be installed to a limited user tmp dir. Maybe. Just thinking out loud here.